recover_affected_workloads.md

```diff
@@ -8,25 +8,31 @@ nav_order: 8
 
 In kubernetes world there is no concept of rescheduling. When pods are bounded to a
 node they run till termination. In a case of hardware failure, or some other
-resource problem that hang the OS, there is no built-in solution to migrate or
+resource problem that hangs the OS, there is no built-in solution to migrate or
 restart existing pods.
 
 If the node misbehaves, and processes can not run anymore the control
 plane will start counting down. It will either be the kubelet process which doesn't
-report status to the control plane [(see node_status_update_frequency)][1] or
+report status to the control plane (see [node_status_update_frequency][]) or
 the containers themselves will fail/crash and be marked as `phase: failed`.
 Nodes status conditions, set by the node-controller in the control plane, will
 reflect the current status and how long the node has been in that state.
 For example, in the case where the node fails to update its status to the
 masters the node condition will be set to `type:Ready status:Unknown`.
 
-In order to gain the workload back, reschedule it and start it all over, the
-control plane needs to know that the running node is gone and the only way to achieve
-that is by deleting the node object, which will trigger [scheduling the pods for
-deletion][2]
+In order to gain the workload back, reschedule it and start it all over, the control plane
+needs to know that the the workloads's pod is not running anymore. There are two
+strategies to accomplish that:
 
-The job of [self-node-remediation](/remediation/self-node-remediation/self-node-remediation/) is to reboot a failing host, delete its node object,
-,and recreate it to restore the node to a working state.
+- [`OutOfServiceTaint`][] - new core Kubernetes feature named [Non-Graceful Node
+  Shutdown], it triggers pods on the node to be forcefully deleted if there are no
+  matching tolerations on the pods. Persistent volumes attached to the shutdown node will
+  be detached, and new pods will be created successfully on a different running node. This
+  is the recommended strategy.
+
+- `ResourceDeletion` - it deletes the pods and detaches associated volumes on the node. This strategy
```

Comment:
this sounds like detaching volumes is something we explicitly do, but we don't (anymore)...?